elements/phaseindicator.lua

```diff
@@ -39,7 +75,7 @@ local function Update(self, event, unit)
 		element:PreUpdate()
 	end
 
-	local isInSamePhase = UnitInPhase(unit) and not UnitIsWarModePhased(unit)
+	local isInSamePhase = not UnitPhaseReason(unit)
```

Comment:
I assume this is new in 9.0, and the name of the function doesn't seem to correlate with the return name, I'd expect that to return something else than a boolean. Am I correct in this assumption?

Reply:
Yes it returns a ["PhaseReason"](https://github.com/Gethe/wow-ui-source/blob/1ceff078c8866706d2134e5329e1043523f10f73/AddOns/Blizzard_APIDocumentation/UnitDocumentation.lua#L1419-L1432) or nil.

Reply:
Still messes me up that 0 evaluates to true in Lua, even after all these years.